cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterStatusService.java

```diff
@@ -401,6 +402,17 @@ public boolean isClusterManagerRunningQuickCheck() {
         }
     }
 
+    @Override
+    public String getClusterManagerVersion() {
```

Comment:
That is a good idea. I now added it as another jira to the "epic", because I think 
- this can become a considerable amount of work
- can be worked on in parallel with other aspects, once this syncer code is on master
- the current implementation already works when the CM is running, the proposal amends functionality
The added jira: https://jira.cloudera.com/browse/CB-13519